include/seqan3/core/concept/cereal.hpp

```diff
@@ -117,7 +117,6 @@ SEQAN3_CONCEPT cereal_text_archive = false;
  *
  * ```
  * #include <seqan3/core/concept/cereal.hpp>
- * using namespace seqan3;
```

Comment:
If you remove this here you also need to add `seqan3::` to the crealisable concept.
This should actually be a real snippet. 
Could maybe remove this change and factor the whole thing out in a follow up PR?

Reply:
Yes, you are absolutely right. I wasn't really looking. I have now adjusted the snippet. Since this changes only 3 more lines, I would leave it in this PR.